style.css

```diff
@@ -78,4 +87,212 @@ body {
   background: #1a2236;
   flex-grow: 1;
   height: 50px;
+  margin: 0 30px 15px 15px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.subGroup {
+  display: flex;
+  flex-direction: column;
+}
+
+#Rectangle21IG {
+  border: 0.33px solid #ffffff1a;
+  background: #ffffff1a;
+  margin: 0 0 0 0;
+}
+
```

Comment:
- [ ] I think it would be better also to adjust the styling of the work and about sections too, to match the Figma design 👍 👍 
   -  Kindly try to fix the [margins and padding](https://www.geeksforgeeks.org/css-margins-padding/) for (the top, bottom, left, and right) for your elements as defined in the design, I have highlighted some for your reference 🙏 
   - Kindly try to match the used colors to those of the Figma design as suggested in the project, Kindly try to focus on small details ✔✔

**Your design** | **Figma design**
| --- | --- |
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/102757126/05a8ac2d-565e-4313-a78f-fc67bfc23bd5) | ![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/102757126/6eba2044-82ee-44ad-ab87-d029f8c24e78)
![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/102757126/99658245-89e4-4dd9-bbfe-c6c9540829bc) | ![image](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/102757126/1ae00756-4ed9-46f9-bb0d-cefb97150436)


## KEEP IT UP :+1: